code/modules/economy/cash.dm

```diff
@@ -17,50 +17,34 @@
 	var/worth = 0
 
 /obj/item/weapon/spacecash/c1
-	name = "1 credip chip"
```

Comment:
Oh yeah, didn't notice that it was excluded. Well then the changes in that file are meaningless anyway. (And that would explain it appearing to be super redundant).